### ISSUE TITLE
How to get SMS metadata

### ISSUE THREAD

**Issue by giautm** [warthog618/modem#4]
Hi, I want to get some metadata from SMS, like `timestamp`.

Currenly, MessageHandle only pass number and message. How to get other information?
```go
	g.StartMessageRx(
		func(number, message string) {
			log.Printf("%s: %s\n", number, message)
		},
		func(err error) {
			log.Printf("err: %v\n", err)
		})
```

**Comment by warthog618** [comment]
Good point - it should also return the corresponding TPDU(s) that you can pull other fields from. I'll have a look at reworking it.

**Comment by warthog618** [comment]
API change committed in cd8b17b.  You might want to give that a try.

**Comment by giautm** [comment]
Thank you for doinng so fastttt. 🥰🥰

**Comment by warthog618** [comment]
Fix released in v0.3.0.
